src/api/index.js

```diff
@@ -108,6 +108,21 @@ export async function getMentors() {
   return [];
 }
 
+export async function getFavorites() {
+  const {_id: userId} = await getCurrentUser();
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/`);
+  if (res.success) {
+    return res.data.mentors.map(mentor => mentor._id);
+  }
+  return [];
+}
+
+export async function addMentorToFavorites(mentorId) {
+  const {_id: userId} = await getCurrentUser();
+  const res = await makeApiCall(`${paths.USERS}/${userId}/favorites/${mentorId}`, {}, 'POST');
```

Comment:
As much as I tested it, it wasn't showed me a blank page (it's just does nothing). The request is already try/catch'ed here: 

https://github.com/Coding-Coach/find-a-mentor/blob/d5228437fb6a13efd2ba81773fa10aca6cc51bb9/src/api/index.js#L30-L46

I changed that now so it will show an error message to the user:

![Screen Shot 2019-11-04 at 15 15 50](https://user-images.githubusercontent.com/3723951/68123554-03d27b80-ff16-11e9-995d-c0c43a78c8e4.png)